src/esn.inbox.libs/i18n/fr.json

```
@@ -5,8 +5,10 @@
   "My identity": "Mon identité",
   "Name or email address": "Nom ou adresse email",
   "New message": "Nouveau message",
+  "Reopen": "Réouvrir",
   "Reopen the composer": "Réouvrir le compositeur",
   "Request a read receipt": "Demander une confirmation de lecture",
+  "This draft has been discarded": "Le brouillon a été effacé",
```

```suggestion
  "This draft has been discarded": "Ce brouillon a été effacé",
```

`le` means the, while `ce` means this.

---

Nice. So the previous translation was also incorrect.

---

Fixed.